unlock-app/src/hooks/usePurchaseKey.ts

```diff
@@ -41,10 +45,16 @@ export const usePurchaseKey = (lock: RawLock, accountAddress: string) => {
             accountAddress,
             transaction.data
           )
+
+          // emit transaction info to main window
+          emitTransactionInfo({
```

Comment:
I wonder if this should eventually move to the `store`?

Reply:
I was thinking of doing that, but didn't want to nest everything too deeply until we had the architecture solidified.